### PR TITLE
Typo corrected in template.tex and change caption from "Listing" to "…

### PR DESCRIPTION
**Pull request by gmsconstantino**
…Listagem" in Portuguese on file fix-babel.clo

### CODE REVIEW

File: LaTeX/unlthesis-files/fix-babel.clo [hunk 1]
```diff
@@ -16,6 +16,7 @@
   \renewcommand{\contentsname}{Índice}%
   \renewcommand{\indexname}{Índice Remissivo}%
   \def\lstlistlistingname{Listagens}%
+  def\lstlistingname{Listagem}%
```
Comment (gmsconstantino) [inline LaTeX/unlthesis-files/fix-babel.clo]: This have a typo, missing \ on def

**Comment by joaomlourenco** [comment]
Constantino, thank you for your proposal of fixes.
The type in `template.tex` is ok.
However, I'm lost with your proposal for `fix-babel.clo`.  Do you propose change "Listagens" to "Listagem"? If this is your proposal I do not agree. Is is a list of listings and not a listing.

**Comment by gmsconstantino** [comment]
With the template as is, in Portuguese when que create a listing the caption will be as shown in image.

image

My proposal on fix-babel.clo is to change the caption to "Listagem".

image

**Comment by joaomlourenco** [comment]
Yep! Got it. Great. Thank you for the fix. :)

**Pull request merged**